source/core/src/main/com/csse3200/game/entities/factories/EnemyFactory.java

```diff
@@ -43,7 +44,7 @@ public class EnemyFactory {
   /**
    * types of enemies
    */
-  private enum EnemyType {
+  public enum EnemyType {
```

Comment:
is there a way to use the enemytype in entity.java if possible since there's duplicates

Reply:
also there's bear enemy type in entity.java but not here is that supposed to happen?

Reply:
Yeah sure, now Enemy NPCs use EnemyType from Entity.java